tests/test_management_commands.py

```diff
@@ -1,21 +1,70 @@
+import json
+from io import StringIO
+
+from django.core.management import call_command
+
 import pytest
+from freezegun import freeze_time
+
+from conference.models import TALK_STATUS
+from tests.common_tools import (
+    create_talk_for_user,
+    get_default_conference,
+    create_user,
+    create_valid_ticket_for_user_and_fare,
+)
+
+pytestmark = pytest.mark.django_db
```

Comment:
this could be replaced by providing `db` fixture to tests.

Reply:
explicit vs implicit and stuff. :)

Reply:
I generally agree - although given that DB is used by what, 80-90% of the tests, I think there's little benefit to marking each test individually whether it depends on the db.